go/vt/vtgate/evalengine/string.go

```diff
@@ -265,3 +270,109 @@ func (builtinRepeat) typeof(env *ExpressionEnv, args []Expr) (sqltypes.Type, fla
 
 	return sqltypes.VarChar, f1
 }
+
+type builtinConv struct {
+}
+
+func (builtinConv) call(env *ExpressionEnv, args []EvalResult, result *EvalResult) {
+	const MaxUint = 18446744073709551615
+	var fromNum uint64
+	var fromNumNeg uint64
+	var isNeg bool
+	var rawString string
+	inarg := &args[0]
+	inarg2 := &args[1]
+	inarg3 := &args[2]
+	fromBase := inarg2.int64()
+	toBase := inarg3.int64()
+	fromNum = 0
+	t := inarg.typeof()
+
+	if inarg.isNull() || inarg2.isNull() || inarg3.isNull() || fromBase < 2 || fromBase > 36 || toBase < 2 || toBase > 36 {
+		result.setNull()
+		return
+	}
```

Comment:
I don't think the checks `inarg2.isNull()` or `inarge3.isNull()` is required. The values of fromBase and toBase would be 0 if they were null respectively, so the condition would still be true.